code/modules/mob/living/living_defines.dm

```diff
@@ -88,4 +88,9 @@
 
 	var/ventcrawl_layer = PIPING_LAYER_DEFAULT
 
-	var/ventcrawl_message_busy
\ No newline at end of file
+	var/ventcrawl_message_busy
+
+	var/taken = FALSE
```

Comment:
What is this var about?

Reply:
That is bad  conflict resolution I removed that var, it was for the offer mob